styles.css

```diff
@@ -0,0 +1,837 @@
+
+  .social-media {
+    display: flex;
+    gap: 24px;
+  }
+
+  .bottom-icon-container {
+    width: 40px;
+    height: 40px;
+    border: 1px solid #dfe1e6;
+    border-radius: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .headline-to-bottom-icon {
+    height: 114px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .bottom-icon-container:hover {
+    background-color: #36b37f;
+  }
+
+  /* WORKS SECTION */
+  #works {
+    width: 81%;
+    display: grid;
+    padding: 80px 0;
+    gap: 48px;
+  }
+
+  #works-title {
+    width: 86%;
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    align-items: center;
+    text-align: center;
+  }
+
+  .works-title-text {
+    font-family: 'Inter', serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+  }
+
+  .indicator {
+    border: 2px solid #36b37e;
+    width: 48px;
+    border-radius: 24px;
+  }
+
+  #work-cards {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    width: 98%;
+  }
+
+  .work-cards-row {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 24px;
+    width: 98%;
+  }
+
+  .card {
+    height: 474px;
+    display: flex;
+    flex-direction: column-reverse;
+    align-items: center;
+    background-image: url("img/Works/Card1.svg");
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+    width: 100%;
+    background-size: 100%;
+  }
+
+  .card-content {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 16px;
+    gap: 16px;
+    background-color: #fff;
+    border-top: none;
+  }
+
+  .card-title {
+    width: 100%;
+    font-family: "poppins", serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  .card-tags {
+    width: 100%;
+    height: 32px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 8px;
+  }
+
+  .card-tag {
+    background: #ebf0ee;
+    height: 32px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .html {
+    width: 51px;
+  }
+
+  .javascript {
+    width: 86px;
+  }
+
+  .css {
+    width: 45px;
+  }
+
+  .ruby {
+    width: 103px;
+  }
+
+  .card-tag-text {
+    font-family: "inter", serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #172b4d;
+  }
+
+  .card-title-button {
+    width: 100%;
+    height: 69px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .card-button {
+    width: 40%;
+    aspect-ratio: 2.54;
+    max-height: 48px;
+    padding: 0 5px;
+    background: #36b37f;
+    border-radius: 4px;
+  }
```

Comment:
- [ ] [OPTIONAL] Please 🙏 make sure to add button 🔘 interactions as mentioned in the [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=cMWNy5D888vYbnt9-0) for increasing user experience 👍 

<img width="408" alt="Screenshot 2023-02-22 at 11 27 01 AM" src="https://user-images.githubusercontent.com/44798044/220541682-6985338c-b89b-462d-a6a6-9915f964f9c8.png">